egi_notebooks_hub/d4science.py

```diff
@@ -63,6 +72,13 @@ class D4ScienceOauthenticator(GenericOAuthenticator):
         help="""The URL for getting DataMiner resources from the
                 Information System of D4science""",
     )
+    d4science_label_name = Unicode(
+        "d4science-namespace",
```

Comment:
this should bevd4science-label?

Reply:
or better d4science-label-name?

Reply:
you mean the default value?

Reply:
yes, is that correct?

Reply:
yes, this is a key-value pair, the key name is defined by this `d4science_label_name` which defaults to `d4science-namespace` and then the value is whatever comes from the authenticator